### PR TITLE
Pull Request for branch 'interactive' that contains functionality for deleting all completed items

### PR DESCRIPTION
**Pull request by jkanyi-web**
- Added functionality for deleting all completed items

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Hi @jkanyi-web , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project with failing linter checks. Kindly fix them and resubmit your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by jkanyi-web** [comment]
Didn't you read my comment? It's not an issue with my code. Try cloning the
project and running all linter checks. Everything passes.

**Comment by chrystalme** [comment]
Hi @jkanyi-web, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that fails the linter test for web-hint. You can contact me on slack to talk more about the error.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by alexander16108** [comment]
Hi @jkanyi-web , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that fails the linter test for lighthouse

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -14,4 +15,6 @@ document.getElementById('formCreate').addEventListener('submit', (e) => {
 document.getElementById('formUpdate').addEventListener('submit', (e) => {
   e.preventDefault();
   updateItem();
-});
\ No newline at end of file
+});
+
+removeCompleted();
```
Comment (alexander16108) [inline src/index.js:23]: as required your project should be able to store items in the local storage after reloading the page. but as I reload the page your items in the array saved in the local storage becomes empty please fix this and try again 👍🏾

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @jkanyi-web , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jkanyi-web** [comment]
Items are already saved in local Storage though. In the requirements, I
haven't read anywhere that required the items to persist on page reload

**Comment by alexander16108** [comment]
@jkanyi-web 
image

**Comment by jkanyi-web** [comment]
Oh thanks for that reminder Alexander, may I request for you to guide me
through this functionality, if you're available?

**Review by cynthiainga — APPROVED** [review]
Hi @jkanyi-web, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] No linters errors ✔️ 
- [ ] Correct GitHub flow ✔️ 
- [ ] Good design ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**